main.py

```diff
@@ -1,10 +1,47 @@
+#!/usr/bin/env python
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
+import json
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_data in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            defaults={"description": player_data["race"].get("description")}
+        )
+
+        guild_data = player_data.get("guild")
+        guild = None
+        if guild_data:
+            guild, _ = Guild.objects.get_or_create(
+                name=guild_data["name"],
+                defaults={"description": guild_data.get("description")}
+            )
+
+        skills = []
+        for skill_data in player_data["race"]["skills"]:
```

Comment:
Verify the JSON structure to ensure that the access path `player_data["race"]["skills"]` is correct. If skills are not nested under race, adjust the access path accordingly.